scripts/build-python-client.sh

```diff
@@ -36,4 +36,8 @@ find client/python/armada_client/ -name '*.py' | xargs sed -i 's/from k8s.io/fro
 export PYTHONPATH=${PWD}/client/python
 python3 ${PWD}/client/python/armada_client/gen/event_typings.py
 
-find client/python/armada_client/k8s/io -name '*.pyi' | xargs sed -i 's/import k8s.io/import armada_client.k8s.io/g'
\ No newline at end of file
+find client/python/armada_client/k8s/io -name '*.pyi' | xargs sed -i 's/import k8s.io/import armada_client.k8s.io/g'
+
+# py.typed is required for mypy to find type hints in the package
+# from: https://mypy.readthedocs.io/en/stable/installed_packages.html#making-pep-561-compatible-packages
+touch client/python/armada_client/py.typed
```

Comment:
Do we need this if you included py.typed when you commit?

Reply:
That is true. I will decide on which way to go soon: either remove this script or remove the named file from the repo.